### PR TITLE
Asset names

### PR DESCRIPTION
**Pull request by signedav**
Asset names should be more descriptive.

### CODE REVIEW

File: scr/DM_Asset_210624_02.ili [hunk 1]
```diff
@@ -637,7 +637,7 @@ VERSION "2019-02-15"  =
     /** Autor
      */
     ASSOCIATION Asset_Contact_Autor =
-      AssetA -- {1..*} Asset;
+      AuthoredAsset -- {1..*} Asset;
```
Comment (signedav) [inline scr/DM_Asset_210624_02.ili:640]: Authored ist nicht so gut.

File: scr/DM_Asset_210624_02.ili [hunk 2]
```diff
@@ -649,7 +649,7 @@ VERSION "2019-02-15"  =
     END Asset_Contact_Contractor;
 
     ASSOCIATION Asset_Contact_Supplier =
-      AssetS -- {1..*} Asset;
+      SuppliedAsset -- {1..*} Asset;
```
Comment (signedav) [inline scr/DM_Asset_210624_02.ili:652]:
```suggestion
      Supplied_Asset -- {1..*} Asset;
```

**Review by signedav — COMMENTED** [review]
WAs ist mit Contracted?

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
 # dmAssets
+testas
```
Comment (oesterli) [inline README.md:2]: More specific, please. :-)

File: scr/DM_Asset_210624_02.ili (hunk 1 above)
Comment (oesterli) [reply scr/DM_Asset_210624_02.ili:640]: See version DM_Asset_210630.ili
AssetSupplier 
AssetContractor 
AssetAuthor

**Review by oesterli — COMMENTED** [review]
review finished

**Comment by signedav** [comment]
Ah yes. It's done here https://github.com/oesterli/dmAssets/blob/main/scr/DM_Asset_210630.ili#L634-L655

**Comment by signedav** [comment]
> AssetSupplier, AssetContractor, AssetAuthor

But in my opinion it does not describe it well. An "asset supplier" is a supplier of an asset, but the role should be the asset that is supplied - so I would prefer "SuppliedAssed" etc.

**Pull request closed without merging**